### ISSUE TITLE
Give people the benefit of the doubt while not tone-policing other people.

### ISSUE THREAD

**Issue by malpinder** [computeranonymous/computer#91]
There was some very unpleasant behaviour on the IRC channel recently. A log of what happened can be seen [here](https://gist.github.com/Ketsuban/3c037ed05bcb381e55a6) and  [here](https://gist.github.com/DRMacIver/fe5b2565ae710791f905). The point at which things start to escalate occurs at 20:29, on line 159.

The first problem is that the question "as a white guy, am I welcome" is being asked. In my experience this is solely asked in a 'waah diversity has gone too far' way: however, I have been informed that this question occurs with good intentions reasonably frequently, and simply saying that they are welcome works fine. I propose adding a line somewhere on the website stating as much: hopefully it will reassure people such that the question stops being asked (and organisers stop having to answer it), and also it will give us a default response of "go read the website".

The second problem is the tone-policing that occurred towards others after they responded to the initial question. As far as I am concerned, the response they received in the IRC channel - from various parties - was unacceptable. It seems the main source of grievance was that they had overstepped an unspoken bound of how 'nice' one should be to people. I think had people in the channel simply ignored the initial comment it would never have escalated the way it did; and put bluntly, I would rather the community alienated overrepresented groups than tone-policed underrepresented ones. In general: if you want to give a person the benefit of the doubt, do so, but do not expect or ask that others do the same.

TLDR; 1: add something on the website that explicitly says 'white cis men are welcome', 2: if someone says something possibly trollish and someone replies to that in a way you think is 'unwelcoming' or 'impolite', ignore it.

(There is obviously a major caveat for number 2, in that if someone says something discriminatory or aggressive it's fine to bring them up on it.)

I am happy to submit a pull request for number 1, if the consensus is that it is a good idea.

**Comment by tef** [comment]
for the record
- numerous people have asked me in private if they should come, because they feel they'll make the place less diverse
- it's been a long running concern that this will end up being all men and fail abjectly on the first meetup

**Comment by DRMacIver** [comment]
The notion I've been mentally using is that "diversity" isn't about percentages, it's about number of people who would otherwise feel marginalised who feel comfortable coming. The rest are welcome if they behave themselves, they just don't contribute to the diversity metric one way or another.

**Comment by joewreschnig** [comment]
I am a white cis man. Never for one second did I think I was not welcome _because of that_. In fact, if I see an organization talking about diversity with a "white cis men welcome" disclaimer it sounds like coddling and I instantly become suspicious of how safe and diverse a space it actually is. So I am ambivalent to suggestion 1, because I think it's salve for a symptom rather addressing the underlying condition.

I don't like the language of "contribute to the diversity metric." Diversity is never lessened by more people coming, unless those people harbor attitudes hostile towards diversity. There is no objective measure of diversity, and no universal "diversity metric." That "metricized" thinking is what leads to the question drags asked in the first place, and is often what creates the appearance of straight/white/cis/able-bodied/male/etc.-exclusion in the first place. If people are thinking this way, we should help them discard flawed ideas of "diversity" as much as reassure them they are welcome.

I'm stilling mulling over 2. Obviously, from what I said in those logs (I'm piman_) I think it's a good idea. But I also worry "easy" formalizations quickly become tools for the privileged. Sorry, I don't have specifics yet, I'm a slow thinker when it comes to these issues.

**Comment by DRMacIver** [comment]
Yes, I agree with you about the metric language. Sorry. That was poorly phrased on my part and I think your phrasing of "Diversity is never lessened by more people coming unless those people harbor attitudes hostile towards diversity" is a much better way of looking at it. 

RE not catering specifically to white cis men: How about a catch-all term in the imposters section to the tune of "If you're worried about whether it's appropriate for you to come for any other reason, come"

Obviously there's a the danger of "any other reason" being interpreted as also meaning "it's cool if you're an MRA douche and want to come here", but hopefully the fact that we immediately follow with a code of conduct will offset that.

**Comment by public** [comment]
I broadly agree with @joewreschnig and @DRMacIver but I also think we definitely shouldn't have language that is explicitly there to make an already well served demographic feel better about themselves.

@DRMacIver I think you are right to worry about that danger. To me your wording reads a bit like it ignores the concerns of people who are genuinely worried about the conduct of people at meetups.

**Comment by timcowlishaw** [comment]
One thing that occurs to me is that it is far easier for those of us who are comparatively privileged to 'give others the benefit of the doubt' when their language or conduct  appears hostile than it is for members of marginalised groups.

With that in mind, should the onus be on the more privileged among us to respond to potential concern trolls (and to educate people with genuine questions), or might this risk crowding out marginalised voices?

Am conflicted about this, but happy to help in any way that is productive!

**Comment by joewreschnig** [comment]
@DRMacIver Without endorsing (or discarding) the exact phrase you wrote, I think you've found exactly the right place/way in the document to address this concern.

Perhaps by analogy with

> If you're from a background which isn't well-represented in most geeky meetups, come along – we want to make a difference.

it could also say

> If you're from a background which is well-represented in most geeky meetups, come along – we want your help making a difference.

**Comment by malpinder** [comment]
+1 for that phrasing. I dislike the idea of coddling, too, but I'm not as good at wording things. :)

**Comment by public** [comment]
> With that in mind, should the onus be on the more privileged among us to respond to potential concern trolls (and to educate people with genuine questions), or might this risk crowding out marginalised voices?

@timcowlishaw IMO absolutely not. This is implicitly saying that the concerns of "marginalised groups" are less valid. If someone reacts with hostility to an apparent concern troll you should avoid tone policing them. (Although sometimes everyone needs to step away from the keyboard for a bit.)

If you feel like explaining that we welcome anyone who wants to play by the rules to people who are asking those sorts of questions go ahead, but I don't think you should feel a need to because of your privilege, there are plenty of different people capable of doing it. The line between being an ally and silencing people through [White Knighting](http://geekfeminism.wikia.com/wiki/White_Knighting) is often narrow.

**Comment by timcowlishaw** [comment]
@public thanks a lot for the clarification, this was precisely my concern.

**Comment by timcowlishaw** [comment]
Also :thumbsup: to @joewrechnig's proposed amends.

**Comment by janepipistrelle** [comment]
+1 for the rephrasing above.

I don't know how to quote, but this: 

"The second problem is the tone-policing that occurred towards others after they responded to the initial question. As far as I am concerned, the response they received in the IRC channel - from various parties - was unacceptable."

and this:

"put bluntly, I would rather the community alienated overrepresented groups than tone-policed underrepresented ones"

@timcowlishaw My approach is to call out bad behaviour including tone-policing/derailing when I see it (the latter by going "nope, that is not an overreaction". idk if that is the best thing, though–it's obviously context dependent–I want to support people not silence them)

**Comment by tef** [comment]
I think this issue can be closed, so @rspinder are you happy with this? I'm cleaning up the bugtracker. I think the current rules, description have been improved enough to warrant closing this issue.

**Comment by malpinder** [comment]
I don't think we reached a consensus on point 2, but since nothing concrete can really come out of it anyway, I'm happy for this to be closed.
